root/components/medium.tt

```diff
@@ -48,7 +48,12 @@
     </td>
     <td>
       [%- link_recording(track.recording, 'show', track.name) -%]
-      <br/>
+      [% IF track.artist_credit_id != track.recording.artist_credit_id %]
+          <div class="small">
+          [% l('Recording artist:') %]
```

Comment:
What about reusing the `Recording by {artist}` string we have in the recording header?

Reply:
Both would probably work, but I feel (since it's not under a recording but a track) "Recording artist: " is slightly more clear. I can also see the usefulness of reusing a string though. @yvanzo: what do you think?
Both options: 
![Screenshot from 2019-10-03 20-33-19](https://user-images.githubusercontent.com/1069224/66151487-165c4b00-e620-11e9-9eef-6716e13f018e.png)
![Screenshot from 2019-10-03 20-37-14](https://user-images.githubusercontent.com/1069224/66151488-165c4b00-e620-11e9-8091-8b99403c17e7.png)

Reply:
I can see the argument that "Recording by ... " right under the track could (wrongly) hint that the above name is the recording name, though it seems unlikely to cause confusion in practice since it's in a track listing (which can be expected to lists tracks). But I'm fine with "Recording artist:" if you think it's more clear.